run_server_test.py

```diff
@@ -399,6 +399,11 @@ def full_run(base_url, remote):
                  {'post': True},
                  as_user='LilyOfTheWest')
 
+
+    resp = fetch('juror: get votes stats',
+                 '/juror/round/%s/votes-stats' % round_id,
+                 as_user='Slaporte')
```

Comment:
did you want to check the response format at all? Just so it doesn't diverge too much from what the angular expects?